packages/i18n/src/locale/bn.json

```diff
@@ -0,0 +1,30 @@
+{
+    "code": "bn",
+    "messages": {
+        "alpha": "এই {field} ক্ষেত্রে কেবলমাত্র অক্ষর থাকতে পারে",
+        "alpha_num": "এই {field} ক্ষেত্রে কেবলমাত্র অক্ষর, সংখ্যা থাকতে পারে",
+        "alpha_dash": "এই {field} ক্ষেত্রে কেবলমাত্র অক্ষর, সংখ্যা, ড্যাশ এবং আন্ডারস্কোর থাকতে পারে",
+        "alpha_spaces": "এই {field} ক্ষেত্রে কেবলমাত্র অক্ষর, সংখ্যা, ড্যাশ এবং আন্ডারস্কোর থাকতে পারে",
+        "between": "এই {field} ক্ষেত্রটি 0:{min} এবং 1:{max} এর মধ্যে হতে হবে",
+        "confirmed": "এই {field} ক্ষেত্রটি মিলছে না",
+        "digits": "এই {field} ক্ষেত্রটি সংখ্যা হতে হবে এবং ঠিক 0:{length} অঙ্ক থাকতে হবে",
+        "dimensions": "এই {field} ক্ষেত্রটি 0:{width} পিক্সেল 1:{height} পিক্সেল হতে হবে",
+        "email": "এই {field} ক্ষেত্রটি একটি বৈধ ইমেল হতে হবে",
+        "excluded": "এই {field} ক্ষেত্রটির মান বৈধ না",
+        "ext": "এই {field} ক্ষেত্রটির ফাইল বৈধ না",
+        "image": "এই {field} ক্ষেত্রটি একটি চিত্র হতে হবে",
+        "integer": "এই {field} ক্ষেত্রটি পূর্ণসংখ্যা হতে হবে",
+        "length": "এই {field} ক্ষেত্রটি 0:{length} দীর্ঘ হতে হবে",
+        "max_value": "এই {field} ক্ষেত্রটি 0:{max} বা তার চেয়ে কম হতে হবে",
+        "max": "এই {field} ক্ষেত্রটি 0:{length} অক্ষরের চেয়ে বেশি হওয়া উচিত নয়",
+        "mimes": "এই {field} ক্ষেত্রের একটি বৈধ ফাইল প্রকার থাকতে হবে",
+        "min_value": "এই {field} ক্ষেত্রটি 0:{min} বা তার বেশি হতে হবে",
+        "min": "এই {field} ক্ষেত্রটি কমপক্ষে 0:{length} অক্ষর হতে হবে",
+        "numeric": "এই {field} ক্ষেত্রে কেবলমাত্র সংখ্যা থাকতে পারে",
+        "one_of": "এই {field} ক্ষেত্রটির মান বৈধ না",
+        "regex": "এই {field} ক্ষেত্রটির বিন্যাস বৈধ না",
+        "required_if": "এই {field} ক্ষেত্রটি অবশ্যক",
+        "required": "এই {field} ক্ষেত্রটি অবশ্যক",
+        "size": "এই {field} ক্ষেত্রের আকার 0:{size}KB এর চেয়ে কম হওয়া উচিত"
+    }
+}
```

Comment:
Thank you for the PR. I will merge it but there is a small issue. we follow `2 spaces` for indentation, here you have `4 spaces`. This is to make sure it's consistent with other files.

Reply:
I will do it myself then